### ISSUE TITLE
Manual definition of classes for new dataset

### ISSUE THREAD

**Issue by MalekSamet** [boschresearch/ALDM#12]
For definition of classes for a new dataset, do I need a dataset_info.json file as for cityscapes? 
Should the new dataloader has the structure of cityscapes.py or ade20k.py? Note that my new dataset has some classes from ade20 and some new classes.

**Comment by YumengLi007** [comment]
Yes, you would need a similar structure of cityscapes.py or ade20k.py
conver_labels are not necessary, it's just for mapping original ADE20K labels to the actual trained IDs. The important thing is to have a consistent and unified defination of class names and class ids for both generator and discriminator.
btw, I have updated README [here](https://github.com/boschresearch/ALDM?tab=readme-ov-file#training)  recently, hopefully it also helps a bit :)
